content/posts/why-meson/index.md

```diff
@@ -0,0 +1,250 @@
++++
+"author" = "ret2libc"
+"title" = "Why Meson"
```

Comment:
What about a more verbose title? "Why we chose Meson as our..."
I am okay with both. But consider it